src/summary/documentation/dimensions-card-row.component.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import styles from "./dimensions-card-row.css";
+
+export default function DimensionsSectionCard(
+  props: DimensionsSectionCardProps
+) {
+  return (
+    <div className={styles.gridContainer}>
```

Comment:
@Andu033 what does this component do? isn't the table enough?

Reply:
Deleted that too